### PR TITLE
feat: [FFM-2194]: Added fix for access control banner not showing

### PR DESCRIPTION
**Pull request by chrisblakely01**
##### Summary:

Fixes an issue where the Developer limit banner doesn't show under certain circumstances under the Access Control page 

##### Jira Links:

https://harness.atlassian.net/browse/FFM-2194

##### Screenshots:

Screenshot 2022-01-28 at 09 17 41

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by chrisblakely01** [comment]
retrigger titlecheck

File: src/modules/20-rbac/components/DeveloperLimitBanner/DeveloperLimitBanner.tsx [hunk 1]
```diff
@@ -25,8 +25,7 @@ const DeveloperLimitBanner = (): ReactElement => {
   const license = useLicenseStore()
 
   const isPaidTeamOrEnterpriseSubscription =
-    (license.licenseInformation.CF?.edition === 'ENTERPRISE' || license.licenseInformation.CF?.edition === 'TEAM') &&
-    license.licenseInformation.CF?.licenseType === 'PAID'
+    license.licenseInformation.CF?.edition === 'ENTERPRISE' || license.licenseInformation.CF?.edition === 'TEAM'
```
Comment (crystal-fong) [inline src/modules/20-rbac/components/DeveloperLimitBanner/DeveloperLimitBanner.tsx:28]: does PAID no longer exist anymore?

**Review by crystal-fong — APPROVED** [review]
nice one 👍

File: src/modules/20-rbac/components/DeveloperLimitBanner/DeveloperLimitBanner.tsx (hunk 1 above)
Comment (chrisblakely01) [reply src/modules/20-rbac/components/DeveloperLimitBanner/DeveloperLimitBanner.tsx:28]: it does - its just the limits apply to the account/license, regardless if there in a free trial or paid license. This was clarify in meeting with Ethan yesterday

**Review by knagurski — APPROVED** [review]
I am unoffended by this change

**Pull request merged**